inoculate/src/trace.rs

```diff
@@ -0,0 +1,168 @@
+use crate::term::{style, ColorMode, OwoColorize};
+use heck::TitleCase;
+use std::fmt;
+use tracing::{field::Field, Event, Level, Subscriber};
+use tracing_subscriber::{
+    field::Visit,
+    fmt::{format::Writer, FmtContext, FormatEvent, FormatFields, FormattedFields},
+    registry::LookupSpan,
+};
+
+#[derive(Default)]
+pub struct CargoFormatter(());
+
+impl<S, N> FormatEvent<S, N> for CargoFormatter
+where
+    S: Subscriber + for<'a> LookupSpan<'a>,
+    N: for<'a> FormatFields<'a> + 'static,
+{
+    fn format_event(
+        &self,
+        ctx: &FmtContext<'_, S, N>,
+        mut writer: Writer<'_>,
+        event: &Event<'_>,
+    ) -> fmt::Result {
+        let metadata = event.metadata();
+        let level = metadata.level();
+        let color = ColorMode::default().should_color_stdout();
```

Comment:
Why the change to `should_color_stdout()` instead of `should_color_stderr()`? I suppose the old cargo formatter used `stderr` and tracing logs to `stdout`?

Reply:
tracing logs to stdout by default; we should probably change it to stderr for consistency with cargo --- i'll do that now